main_2024.py

```diff
@@ -357,7 +400,7 @@ def main() -> int:
                 return -1
 
         try:
-            geolocation_data = geolocation_to_main_queue.queue.get_nowait()
+            geolocation_data = geolocation_to_cluster_estimation_queue.queue.get_nowait()
```

Comment:
We do not want to log geolocation in the main logger as it has its own logging now. You can get rid of the entire try block.